cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js

```diff
@@ -1,5 +1,9 @@
 angular.module(PKG.name + '.feature.admin').controller('AdminNamespaceDatasetMetadataController',
+<<<<<<< HEAD
```

Comment:
this is strange... Let me check one more time. Those should be done by "git checkout --theirs .."